Makefile

```diff
@@ -211,16 +214,16 @@ docker-push:
 release-manifest:
 	@sed -i -e 's/^VERSION := .*/VERSION := ${NEWVERSION}/' ./Makefile
 	@sed -i'' -e 's@image: $(REPOSITORY):.*@image: $(REPOSITORY):'"$(NEWVERSION)"'@' ./config/manager/manager.yaml ./manifest_staging/deploy/gatekeeper.yaml
-	@sed -i "s/appVersion: .*/appVersion: ${NEWVERSION}/" ./manifest_staging/chart/gatekeeper-operator/Chart.yaml
-	@sed -i "s/version: .*/version: ${NEWVERSION}/" ./manifest_staging/chart/gatekeeper-operator/Chart.yaml
-	@sed -i "s/release: .*/release: ${NEWVERSION}/" ./manifest_staging/chart/gatekeeper-operator/values.yaml
-	@sed -i "s@repository: .*@repository: ${REPOSITORY}@" ./manifest_staging/chart/gatekeeper-operator/values.yaml
+	@sed -i "s/appVersion: .*/appVersion: ${NEWVERSION}/" ./manifest_staging/helm/gatekeeper/Chart.yaml
+	@sed -i "s/version: .*/version: ${NEWVERSION}/" ./manifest_staging/helm/gatekeeper/Chart.yaml
+	@sed -i "s/release: .*/release: ${NEWVERSION}/" ./manifest_staging/helm/gatekeeper/values.yaml
+	@sed -i "s@repository: .*@repository: ${REPOSITORY}@" ./manifest_staging/helm/gatekeeper/values.yaml
 
 promote-staging-manifest:
 	@rm -rf deploy
 	@cp -r manifest_staging/deploy .
-	@rm -rf chart
-	@cp -r manifest_staging/chart .
+	@rm -rf helm
```

Comment:
can we keep the `chart` folder as helm is the binary not the chart itself.

Reply:
Is there some standard I can reference. I tried looking and couldn't find any. I did see some repos using a `helm` directory (prometheus-operator before it migrated to the now-deprecated chart repo), but didn't get a hint of substructure.

TL;DR, I have no opinion about what the name should be, other than I'd like to know that I'm following some convention, b/c it appears we may not have been doing so originally and now is the time to fix that.

Reply:
I've also see `charts` -- plural

Reply:
Usually top level directory convention is `chart` or `charts` (for multiple charts) afaik

Reply:
Renamed directory to `charts` per the mtg

Reply:
did you want to also update this to `@rm -rf charts`

Reply:
So I did, thanks!